src/ripple/app/main/DBInit.cpp

```diff
@@ -25,6 +25,9 @@ namespace ripple {
 // Transaction database holds transactions and public keys
 const char* TxnDBInit[] =
 {
+    "PRAGMA journal_mode=OFF",
+    "PRAGMA page_size=65536;",
+    "VACUUM;",
```

Comment:
`VACUUM` would be bad here...I think we just want the `page_size` in both inits.

Reply:
@mellery451 Good catch. For some reason I had problems initially setting the page_size and (wrongly) assumed that this was only called upon DB creation rather than every time rippled starts. Anyway, the placement of setting page_size seems to matter for it to take effect immediately upon database creation, so I put it at the beginning.